### ISSUE TITLE
Store `mergeable` in the patches table

### ISSUE THREAD

**Issue by notriddle** [bors-ng/bors-ng#853]
This is an optimization on top of https://github.com/bors-ng/bors-ng/pull/843

It would be nice if we didn't have to hit the repository every time we split a batch. It's not a big enough deal to block on, but it makes everything just a bit slower, so it would be nice to avoid it.

**Comment by SocalNick** [comment]
Doesn't merge-ability change as additional batches are landed on the trunk branch? cc @noizwaves @ansonlc

**Comment by noizwaves** [comment]
@SocalNick I haven't looked into it yet, but I suspect we would get updates (via one of the webhooks) that contains this information. Highly likely when the PR changes, but I'm unsure if changes to trunk branch triggers new PR specific webhooks.

**Comment by notriddle** [comment]
> I haven't looked into it yet, but I suspect we would get updates (via one of the webhooks) that contains this information.

Yeah, we would need to use the `pull_request` webhook (GitHub uses the `synchronize` action whenever the relevant branches change).
